src/styles/drawer.css

```diff
@@ -135,3 +135,13 @@
   padding: 2em 2em .5em;
   margin: 0;
 }
+
+[role=banner] .coz-drawer--settings {
+  padding-bottom: env(safe-area-inset-bottom);
+}
+
+/* /!\ Trick to prevent application from scrolling in the background when the drawer is opened */
+[role=banner][data-drawer-visible=true] + [role=application] {
+  position: fixed;
+  width: 100%;
```

Comment:
Is this styles not breaking applications layout?

Reply:
Not as far as I tested, no. 
The thing is that the drawer cover 90% of the application so you don't have much to see of it.

Reply:
Let's try then